gravis3d/body.py

```diff
@@ -13,7 +13,7 @@ def __init__(self, radius = None, pos = None, velocity = None, colour = None,den
         self.mass = (4/3)*pi*(self.radius**3)*self.density
         self.pos = pos or vector(0,0,0)
         self.velocity = velocity or vector(0,0,0)
-        self.color = colour or color.orange
+        self.color = colour or color.green
```

Comment:
why change of color?

Reply:
Actually i want to have the new merged body of different colour , and for orange might be a good colour to recognise so i coloured the olders body with th green one.